src/api/app/models/issue_tracker.rb

```diff
@@ -32,6 +33,18 @@ def write_to_backend
     self.issues_updated ||= Time.now
   end
 
+  def cve?
+    kind == "cve"
+  end
+
+  def verify_issue_name!(name)
```

Comment:
I can do that, but it makes the code IMHO unnecessary complicate. One needs a tracker object anyway and the issue object can't exist anyway. It would look like

Issue.verify_name!(tracker, name)

and the calling code must create the tracker object before. Now it is just a one liner. Or we add another method for this functionality, but we have basically created three methods for a simple regexp check then ....